Source/RunnerAddOn.cpp

```diff
@@ -237,13 +246,13 @@ populate_menu(BMessage* message, BMenu* menu, BHandler* handler)
 	BMenu* prefsSubMenu = NULL;
 	builder
 		.AddSeparator()
-		.AddMenu("Preferences & Help")
+		.AddMenu(B_TRANSLATE("Settings & Help"))
```

Comment:
I'm still debating this one. There are still quite a few apps that refer to it as "Preferences".  Tracker & Deskbar for example.

Reply:
Right, sorry I missed pointing out this change.

IMO, and I think at least Haiku apps generally adhere to that, "Preferences" are the applets found in Deskbar's  "Preferences" menu, or are system component preferences like for Tracker and Deskbar.
"Application-internal" preferences are "Settings". 

Things become difficult when it comes to "Options". :)
Those are best avoided, but maybe those would be options that refer to the current document or view, and are separate from the "app-internal" Settings. At least that's my current thinking...